src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -158,6 +158,14 @@ internal virtual ImmutableArray<LocalSymbol> GetDeclaredLocalsForScope(CSharpSyn
         }
 
         /// <summary>
+        /// Get local functions declared immediately in scope represented by the node.
+        /// </summary>
+        internal virtual ImmutableArray<LocalFunctionSymbol> GetDeclaredLocalFunctionsForScope(CSharpSyntaxNode node)
```

Comment:
Good point. Right now it serves as basically a debug sanity check, to make sure that the syntax we're in and the binder we have are the same thing. I can remove the parameter if you think that's the right thing to do (as well as GetDeclaredLocalsForScope, which is where I copied this pattern from).